src/SignalR/clients/ts/FunctionalTests/ts/HubConnectionTests.ts

```diff
@@ -838,6 +841,52 @@ describe("hubConnection", () => {
         }
     });
 
+    it("connection id matches server side connection id", async (done) => {
```

Comment:
Do we have a test that verifies when it's null?

Reply:
Theres' an assert before start is called, there could be another for after stop. I'll add another test for when we skip negotiate and the id is always null.